AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/resourcegroup/control/ResourcesScreenQueries.java

```diff
@@ -39,9 +39,10 @@ public Query searchResourceTypeByName(String resourceTypeName) {
 				.setParameter("resourceTypeName", resourceTypeName);
 	}
 	
-	public Query searchResourceByName(String resourceName) {
-		return entityManager.createQuery("from ResourceEntity as res where res.name like :resourceName")
-				.setParameter("resourceName", resourceName);
+	public Query searchResourceByName(String resourceName, Integer id) {
```

Comment:
Maybe this method should be renamed? `searchOtherResourcesWithName` or something?

Reply:
Agreed - I have changed it